cupy/_math/rounding.py

```diff
@@ -29,6 +29,11 @@ def round_(a, decimals=0, out=None):
     return around(a, decimals, out=out)
 
 
+round = around
+round.__name__ = 'round'
+round.__doc__ = around.__doc__.replace('around', 'round')
+
+
```

Comment:
In NumPy implementation, `numpy.round` is just an alias of `numpy.round_` How about defining `round = round_` here?

Reply:
Do you mean just `round = round_`, removing the `__doc__` setting? NumPy's `round` does have a docstring that is minimal and just refers the user to `around`.

Reply:
Can you make a suggestion here with exactly what you want and I can accept it?

Reply:
In CuPy, undocumented aliases are defined like this:

https://github.com/cupy/cupy/blob/v9.0.0b1/cupy/__init__.py#L708-L711

So just adding `from cupy._math.rounding import round_ as round` to `cupy/__init__.py` sounds enough to me.